lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -732,6 +709,11 @@ PROJECTED_ENUMS(PROJECTED_ENUM)
 
     void IntlEngineInterfaceExtensionObject::InjectIntlLibraryCode(_In_ ScriptContext * scriptContext, DynamicObject* intlObject, IntlInitializationType intlInitializationType)
     {
+
+        // Ensure JsBuiltIns are initialized before initializing Intl which uses some of them.
+        scriptContext->GetLibrary()->EnsureArrayBuiltInsAreReady();
+        scriptContext->GetLibrary()->EnsureMathBuiltInsAreReady();
```

Comment:
Having removed the call above these 2 calls are added here to load Array_prototype and Math_object before Intl only when actually loading Intl.

Ideally this dependency would be removed but that would take some more substantial work.